[2023-09-04T06:02:52.210+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2023-09-04T06:02:52.220+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2023-09-04T06:02:52.221+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:02:52.238+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-04-29 00:00:00+00:00
[2023-09-04T06:02:52.246+0000] {standard_task_runner.py:57} INFO - Started process 5544 to run task
[2023-09-04T06:02:52.249+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-04-29T00:00:00+00:00', '--job-id', '529', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpb9z5c79t']
[2023-09-04T06:02:52.251+0000] {standard_task_runner.py:85} INFO - Job 529: Subtask insert_into_table
[2023-09-04T06:02:52.310+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-29T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:02:52.428+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-04-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-29T00:00:00+00:00'
[2023-09-04T06:02:52.430+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-04-29', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:02:52.441+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:52.452+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:52.463+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-04-29', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:02:52.467+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:02:52.480+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220429T000000, start_date=20230904T060252, end_date=20230904T060252
[2023-09-04T06:02:52.502+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:02:52.518+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:05:41.746+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2023-09-04T06:05:41.776+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2023-09-04T06:05:41.777+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:05:41.939+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-04-29 00:00:00+00:00
[2023-09-04T06:05:41.952+0000] {standard_task_runner.py:57} INFO - Started process 6874 to run task
[2023-09-04T06:05:41.958+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-04-29T00:00:00+00:00', '--job-id', '1086', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpdo9dt6sl']
[2023-09-04T06:05:41.963+0000] {standard_task_runner.py:85} INFO - Job 1086: Subtask insert_into_table
[2023-09-04T06:05:42.183+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-29T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:05:42.447+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-04-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-29T00:00:00+00:00'
[2023-09-04T06:05:42.450+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-04-29', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:05:42.484+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:42.549+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:42.595+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-04-29', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:05:42.608+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:05:42.655+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220429T000000, start_date=20230904T060541, end_date=20230904T060542
[2023-09-04T06:05:43.112+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:05:43.162+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
